.github/workflows/client-credential.yml

```diff
@@ -0,0 +1,45 @@
+name: msal-node client-credential Regression Test
+on:
+    pull_request:
+        types: [opened, reopened, synchronize, ready_for_review]
+        paths:
+            - "regression-tests/msal-node/client-credential/*"
+            - "lib/msal-node/**/*"
+            - "lib/msal-common/**/*"
+            - "!**.md"
+            - ".github/workflows/client-credential.yml"
+    merge_group:
+        types: [checks_requested]
+
+permissions:
+    contents: write
+
+jobs:
+  benchmark:
+    name: Run msal-node client-credential Regression Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - name: Run benchmark
+        run: cd regression-tests/msal-node/client-credential && npm install && node index.js | tee output.txt
+
+      - name: Download previous benchmark data
+        uses: actions/cache@v3
+        with:
+          path: ./cache
+          key: ${{ runner.os }}-benchmark
+
+      - name: Store benchmark result
```

Comment:
I'd also add to push results to GitHub pages to enable charts.

- Create a new branch gh-pages with just index.html to store the data for diagram. (Like [this](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/tree/gh-pages), the action will add data files automatically).
- Update repo settings
  - Settings > Pages > Build and deployment, select Deploy from a branch; for branch select gh-pages.
- Update Action yml to upload data. (See [here](https://github.com/benchmark-action/github-action-benchmark#charts-on-github-pages-1) and [example](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/88833791343d4039409d595edbb0cc5f2f42a418/.github/workflows/benchmark-action.yml))

Reply:
done, thanks